core/src/main/java/bisq/core/provider/price/PriceProvider.java

```diff
@@ -70,19 +70,10 @@ public PriceProvider(HttpClient httpClient, String baseUrl) {
 
         // get btc per xmr price to convert all prices to xmr
         // TODO (woodser): currently using bisq price feed, switch?
-        Double btcPerXmr = null;
```

Comment:
i started changing it here but then decided to keep this as it is because it is used in the current desktop UI and map the inverted values just for the API calls.

So the changes in this class does not change any functionality, i just extracted one method and used normal `for` that does not need the final variable